external/gputils/src_lib/test_utils.cu

```diff
@@ -372,6 +372,7 @@ INSTANTIATE_TEMPLATES(unsigned short);
 INSTANTIATE_TEMPLATES(unsigned char);
 INSTANTIATE_TEMPLATES(complex<float>);
 INSTANTIATE_TEMPLATES(complex<double>);
+INSTANTIATE_TEMPLATES(complex<int>);
```

Comment:
Technically speaking, the `INSTANTIATE_TEMPLATES` already provide the trailing semicolons, so the macro calls here should have not semicolon.